wiki/development/arma-3-scheduler-and-our-practices.md

```diff
@@ -10,87 +10,99 @@ order: 8
 ## 1. Terminology
 
 ### 1.1 Frame
-A single rendered frame of Arma 3.
+A single rendered frame of Arma 3 or a single simulation frame for dedicated servers and headless clients.
 
-### 1.2 Scheduled Space
+### 1.2 Scheduled Environment
+Functions and scripts run in the scheduled environment when they are executed by one of the following:
 
-This refers to execution would is ruled by the Arma 3 default script scheduling engine. This would include:
+* `spawn`
+* `execVM`
+* CfgFunctions postInit
+* `call` from a function running in *scheduled* environment
 
-* spawn
-* execVM 
+They can be suspended by `waitUntil` or `sleep`. To check if a function runs in the scheduled environment you can use `canSuspend`.
 
-### 1.3 Unscheduled Space
-This refers to execution which is linear; what this means is that the code will run to completion prior to executing the current frame. It must complete, but is guaranteed to run within a given frame.
+### 1.3 Unscheduled Environment
+Code running in the unscheduled environment uses linear execution, that means it will always run from start to finish within one frame, without any pauses. The following will always be executed in the unscheduled environment:
 
-* perFrameHandler
-* Extended_EventHandlers
-* addEventHandler
+* Event Handlers
+* CBA Per Frame Handlers (PFHs)
```

Comment:
As much as I know they can have apostrophes, though it's more usual that they don't have them.